versions/development/SPEC.md

```diff
@@ -3008,6 +3009,8 @@ Example 1:
   String chocolove = sub(chocolike, "like", "love") # I love chocolate when it's late
   String chocoearly = sub(chocolike, "late", "early") # I like chocoearly when it's early
   String chocolate = sub(chocolike, "late$", "early") # I like chocolate when it's early
+  String chocoearlylate = sub(chocolike, "[^ ]late", "early") # I like chocoearly when it's late
+  String choco4 = sub(chocolike, " [:alpha:]{4} ", " 4444 ") # I 4444 chocolate 4444 it's late
```

Comment:
I realize this has already passed voting, but I briefly looked into how this might be implemented in miniwdl and noticed that there are some typos in these examples. They should look like the following:
```wdl
 String chocoearlylate = sub(chocolike, "[^ ]late", "early") # I like chocearly when it's late
```
The second `o` will also be replaced, due to the `[^ ]`.

```wdl
 String choco4 = sub(chocolike, " [[:alpha:]]{4} ", " 4444 ") # I 4444 chocolate 4444 it's late
```
Character classes require and additional set of `[]`.